minet/twitter/api_scraper.py

```diff
@@ -56,6 +56,10 @@ def forge_search_url(query):
     )
 
 
+def guest_token_API_url():
```

Comment:
This method does not seem to be used anywhere.

Reply:
My bad.

Reply:
This function is back and still unused :)

Reply:
Oh no, it is actually used my bad. Though since it only returns a constant string I think I'd prefer a constant.